### PR TITLE
[14.0][MIG] event_registration_cancel_reason: Migration to 14.0

### PR DESCRIPTION
**Pull request by LudLaf**
Standard migration to version 14.0

@Tecnativa 

TT35392

### CODE REVIEW

File: event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 Antiun Ingeniería S.L.
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import exceptions, fields
+from odoo.tests import common
+
+
+class TestEventRegistrationCancelReason(common.TransactionCase):
```
Comment (rousseldenis) [inline event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py]: You can improve speed using SavepointCase

File: event_registration_cancel_reason/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
```
Comment (rousseldenis) [inline event_registration_cancel_reason/tests/__init__.py:1]: No need of license lines on `__init__`

**Review by rousseldenis — APPROVED** [review]
Code review. Nitpicking

File: event_registration_cancel_reason/tests/__init__.py (hunk 2 above)
Comment (pedrobaeza) [reply event_registration_cancel_reason/tests/__init__.py:1]: Well, it's common practice. What we remove is copyright in these files.

File: event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py (hunk 1 above)
Comment (pedrobaeza) [reply event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py]: Done

**Review by pedrobaeza — APPROVED** [review]

File: event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import exceptions, fields
+from odoo.tests import common
+
+
+class TestEventRegistrationCancelReason(common.SavepointCase):
+    @classmethod()
```
Comment (chienandalu) [inline event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py]:
```suggestion
    @classmethod
```

**Pull request merged**